src/fastfield/multivalued/writer.rs

```diff
@@ -38,16 +38,19 @@ pub struct MultiValuedFastFieldWriter {
     field: Field,
     vals: Vec<UnorderedTermId>,
     doc_index: Vec<u64>,
+    is_storing_term_ids: bool,
     is_facet: bool,
 }
 
 impl MultiValuedFastFieldWriter {
-    /// Creates a new `IntFastFieldWriter`
-    pub(crate) fn new(field: Field, is_facet: bool) -> Self {
+    /// Creates a new `MultiValuedFastFieldWriter`
+    pub(crate) fn new(field: Field, mut is_storing_term_ids: bool, is_facet: bool) -> Self {
```

Comment:
I wonder if it would be less confusing if we added a enum type

```
enum FastFieldType {
   Numerical,
   String,
   Facet,
}

Reply:
yep